bats/tests/containers/volumes.bats

```diff
@@ -0,0 +1,274 @@
+load '../helpers/load'
+
+get_tempdir() {
+    if ! is_windows || ! using_windows_exe; then
+        echo "$BATS_TEST_TMPDIR"
+        return
+    fi
+    # On Windows, create a temporary directory that is in the Windows temporary
+    # directory so that it mounts correctly.  Note that in CI we end up running
+    # with PSModulePath set to pwsh (7.x) paths, and that breaks the code for
+    # PowerShell 5.1.  So we need to have alternative code in that case.
+    # See also https://github.com/PowerShell/PowerShell/issues/14100
+    if command -v pwsh.exe &>/dev/null; then
+        # shellcheck disable=SC2016 # Don't expand PowerShell expansion
+        local command='
+            $([System.IO.Directory]::CreateTempSubdirectory()).FullName
+        '
+        run pwsh.exe -Command "$command"
+        assert_success
+    else
+        # PowerShell 5.1 is built against .net Framework 4.x and doesn't have
+        # [System.IO.Directory]::CreateTempSubdirectory(); create a temporary
+        # file and use its name instead.
+        # shellcheck disable=SC2016 # Don't expand PowerShell expansion
+        local command='
+            $name = New-TemporaryFile
+            Remove-Item -Path $name
+            Start-Sleep -Seconds 1 # Allow anti-virus to do stuff
```

Comment:
We need better documentation here. I'm unsure if I understand the correlation between pausing the execution of the script for 1 second and Anti-virus.

Maybe something like:

_pause for 1 second to allow time for background processes (e.g., antivirus) to complete their operations on the deleted file before creating the directory._